echidna/core/spectra.py

```diff
@@ -229,3 +229,29 @@ def add(self, spectrum):
         self._data += spectrum._data
         self._raw_events += spectrum._raw_events
         self._num_decays += spectrum._num_decays
+
+    def rebin(self, *args):
+        '''rebin spectra data into a smaller spectra of the same rank whose
+        dimensions are factors of the original dimensions.
+        Adapted from http://wiki.scipy.org/Cookbook/Rebinning
```

Comment:
Could you split this into two paragraphs, a short sentence for the first line and then a longer sentence with more explanation. Also capitalise "Rebin"